cmake/OpenCVUtils.cmake

```diff
@@ -796,11 +796,20 @@ macro(ocv_get_all_libs _modules _extra _3rdparty)
   set(${_3rdparty} "")
   foreach(m ${OPENCV_MODULES_PUBLIC})
     get_target_property(deps ${m} INTERFACE_LINK_LIBRARIES)
+    if(NOT deps)
+      set(deps "")
+    endif()
     list(INSERT ${_modules} 0 ${deps} ${m})
     foreach (dep ${deps} ${OPENCV_LINKER_LIBS})
       if (NOT DEFINED OPENCV_MODULE_${dep}_LOCATION)
         if (TARGET ${dep})
-          list(INSERT ${_3rdparty} 0 ${dep})
+          get_target_property(_isimported ${dep} IMPORTED)
```

Comment:
IPP library is added like this:
```.cmake
add_library(ipp${name} STATIC IMPORTED)
<...>
install(FILES ${ipp_lib} DESTINATION ${OPENCV_3P_LIB_INSTALL_PATH} COMPONENT main)
```
Other 3rdparty libraries are normal targets and they are automatically put into _3rdparty_ list because of `OPENCV_MODULE_${dep}_LOCATION` variable absence. But IPP library is imported target and it will be put into _extra_ list without ths check.

It seems that we can use `OpenCVModules_TARGETS` variable to check whether the target is exported, but there will be no IPP library in this list (I guess).